app/controllers/dbhero/application_controller.rb

```diff
@@ -3,7 +3,7 @@ class ApplicationController < ActionController::Base
     def check_auth
       if Dbhero.authenticate
         unless _current_user && call_custom_auth
-          raise ActionController::RoutingError.new('Forbidden')
+          redirect_to '/users/sign_in'
```

Comment:
@bielversallini the problem if use /users/sign_in is because is not everyone have this routes defined.